frontend/packages/citizen-frontend/src/applications/editor/ApplicationFormData.ts

```diff
@@ -116,9 +114,7 @@ export function apiDataToFormData(
       nonCaretakerFirstName: undefined,
       nonCaretakerLastName: undefined,
       nonCaretakerSSN: undefined,
-      knownSiblings: application.form.otherChildren,
-      hasOtherSiblings: false,
-      otherChildren: []
+      otherChildren: application.form.otherChildren
```

Comment:
Lets discuss this next week, but I'm thinking we might want to default these above to empty strings and remove all question marks from the type.